projects/plugins/jetpack/modules/post-list.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Module Name: Post List
+ * Module Description: Enhance the classic view of the Admin section of your WordPress site.
+ * Sort Order: 31
+ * Recommendation Order: 12
+ * First Introduced: $$next-version$$
```

Comment:
I don't believe this will work, the script does not support "Fist introduced" yet:
https://github.com/Automattic/jetpack/blob/b3179d4347dcf73269985e1801b61bf53fbc441a/tools/replace-next-version-tag.sh#L16-L20